post-processor/googlecompute-import/post-processor.go

```diff
@@ -0,0 +1,204 @@
+package googlecomputeimport
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/storage/v1"
+
+	"github.com/hashicorp/packer/builder/googlecompute"
+	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/helper/config"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/post-processor/compress"
+	"github.com/hashicorp/packer/template/interpolate"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/jwt"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+
+	Bucket            string            `mapstructure:"bucket"`
```

Comment:
Maybe there should be something like: https://www.packer.io/docs/post-processors/amazon-import.html#s3_key_name

Reply:
Partially fixed. I added a new config option `gcs_object_name`, but the time stamp in the default value is not being interpolated. I'm getting a file literally named `packer-import-{{timestamp}}.tar.gz` in the GCS bucket. I'll need to sort this out.